server/halomod_app/__init__.py

```diff
@@ -191,14 +166,14 @@ def get_plot_data():
                 data["xs"] = list(xs[mask])  # apply mask and save xs into data dict
             except Exception as e:
                 abort(
-                    400, f"Error encountered getting {fig_type} for model {name}. {str(e)}.")
-                print(f"Error encountered getting {fig_type} for model {name}")
+                    400, f"Error encountered getting {y_param} for model {name}. {str(e)}.")
+                print(f"Error encountered getting {y_param} for model {name}")
                 print(e)
 
             res["plot_data"][name] = data  # put data in response object
 
         # put figure metadata into response
-        res["plot_details"] = utils.KEYMAP[fig_type]
+        res["plot_details"] = utils.KEYMAP[y_param]
```

Comment:
Can all references to KEYMAP be changed now? This might make a big impact on tests, because this will show if the references to the globals files is actually working in all cases on the Python side. 

Also if KEYMAP is no longer needed, then it can probably be deleted so we don't have the same data in two places.

Reply:
Well kindof... 
other endpoints are still using it. I did not want to refactor those as I had no idea what they did.

Reply:
this is removed now.